test/fit_tests/common/fit_common.py

```diff
@@ -363,7 +363,7 @@ def restful(url_command, rest_action='get', rest_payload=[], rest_timeout=None,
                                        verify=sslverify,
                                        )
         if rest_action == "binary-put":
```

Comment:
rather than track the content-headers in a common library here, why not use the swagger generated python libraries that map directly to the swagger definition file? It would remove this sort of issue from the future, as well as give you immediately expanded scope for actions to be used in test cases.